test/lib/navigation.rb

```diff
@@ -0,0 +1,18 @@
+require 'page_navigation'
+
+module OllertTest
+  module Navigation
+    include PageNavigation
+
+    def on(cls, &block)
+      screen = cls.new
+      block.call screen if block
```

Comment:
i think i'd prefer to see an `unless block.nil?` here instead of relying on truthiness.